test/reporters/base.spec.js

```diff
@@ -443,4 +443,40 @@ describe('Base reporter', function() {
       sinon.restore();
     });
   });
+
+  describe('When there is StackTrace in the error message', function() {
+    var baseConsoleLog;
```

Comment:
You can use `let` instead of `var` here now

Reply:
i got it!